src/App.jsx

```diff
@@ -1,60 +1,144 @@
-import "./App.css"
-
+import "./App.css";
+import { useState } from "react";
 function App() {
+  const [calculation, setCalculation] = useState("");
+  const [leftInput, setLeftInput] = useState("");
+  const [rightInput, setRightInput] = useState("");
+  const [operand, setOperand] = useState("");
+  const [storedResult, setStoredResult] = useState("");
+  function numbersButtons(type) {
+    let numbers = [];
+    for (let i = 1; i <= 12; i++) {
+      if (i === 10) {
+        numbers.push(0);
+        continue;
+      }
+      if (i === 11) {
+        numbers.push("Clear");
```

Comment:
don't include clear and full stop buttons because you will find it harder to assingn the `onClick` handlers

better to just define a const numbers array ["1", "2", ...] and map through this